ci-bench/src/benchmark.rs

```diff
@@ -27,40 +26,13 @@ pub fn validate_benchmarks(benchmarks: &[Benchmark]) -> anyhow::Result<()> {
         );
     }
 
-    // Detect dangling benchmark references
-    let all_names: FxHashSet<_> = benchmarks
-        .iter()
-        .map(|b| b.name.as_str())
-        .collect();
-    let referenced_names: FxHashSet<_> = benchmarks
-        .iter()
-        .flat_map(|b| match &b.reporting_mode {
-            ReportingMode::AllInstructions => None,
-            ReportingMode::AllInstructionsExceptSetup(name) => Some(name.as_str()),
-        })
-        .collect();
-
-    let undefined_names: Vec<_> = referenced_names
-        .difference(&all_names)
-        .cloned()
-        .collect();
-    if !undefined_names.is_empty() {
-        anyhow::bail!("The following benchmark names are referenced, but have no corresponding benchmarks: {}",
-            undefined_names.join(", "));
-    }
-
     Ok(())
 }
 
 /// Specifies how the results of a particular benchmark should be reported
 pub enum ReportingMode {
     /// All instructions are reported
     AllInstructions,
-    /// All instructions are reported, after subtracting the instructions of the setup code
```

Comment:
Is the enum still useful?

Reply:
removed and squashed removal into the original commit